### ISSUE TITLE
Genetonic unable to install 

### ISSUE THREAD

**Issue by fishmatt** [federicomarini/GeneTonic#54]
Hi,

I have used the package before and love it but this was on a different computer. I am struggling to install the package because of this error:
Error: object ‘universal’ is not exported by 'namespace:backbone'
Backbone package I have is the latest version 2.1.2

Is there a different version of this package i need to use? 

Thanks!

**Comment by federicomarini** [comment]
Can you post the output of `sessionInfo`? It can be that there is an older version of `backbone` installed and that is not compatible with the newer GeneTonic.
Try reinstalling the latest `backbone` (even if already present on that machine)?

**Comment by fishmatt** [comment]
Yes sure. Have attached session info. I have tried reinstalling backbone but no difference. Thanks for the help!

`sessionInfo()
R version 4.1.2 (2021-11-01)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 22.04.2 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.10.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.10.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8   
 [6] LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] BiocManager_1.30.21 compiler_4.1.2      tools_4.1.2         rstudioapi_0.14    `

**Comment by fishmatt** [comment]
Update- I tried my mac system to see I got the same problem, which I did
I uninstalled backbone and installed the previous version 2.1.0.  I loaded the package and gene tonic successfully. 

`R version 4.1.3 (2022-03-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Monterey 12.5.1

Matrix products: default
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_GB.UTF-8/en_GB.UTF-8/en_GB.UTF-8/C/en_GB.UTF-8/en_GB.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] GeneTonic_1.6.4     backbone_2.1.0      Biobase_2.54.0      BiocGenerics_0.40.0
`

**Comment by federicomarini** [comment]
Well, probably the best option is to work with a more recent of R/Bioc/GeneTonic itself.
This would make sure you are with the newest features, and the setup works.
